src/main/java/com/sda/zdjavapol107/memorki/Main.java

```diff
@@ -0,0 +1,149 @@
+package com.sda.zdjavapol107.memorki;
+
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+    public static String[][] board = new String[4][4];
+    public static String[][] elements = new String[4][4];
+    public static Scanner sc = new Scanner(System.in);
+
+    public static void boardPrinter() {
+        for (int i = 0; i < 4; i++) {
```

Comment:
nie uzywaj takich glowych liczb jesli to sa stale, stworz pole static final :) czytelniejsze bedzie BOARD_SIZE zamiast 4, nie mowiac juz o tym ze gdyby trzeba bylo zmeinic rozmiar planszy robisz to w jednym miejscu :D